File: Problemas/9A - Die Roll/9A_Die_Roll.py
# 9A - Die Roll
# https://codeforces.com/problemset/problem/9/A

# Input:
# 1) yakko = Resultado del dado de Yakko
# 2) wakko = Resultado del dado de Wakko
yakko, wakko = map(int, input().split())

# Calcula el valor del dado que Dot debe superar
valor_maximo = max(yakko, wakko)

# Cantidad de valores que le darán a Dot la victoria
dot = 6 - valor_maximo + 1

# Simplifica la fracción acorde a las especificaciones del problema
if dot == 0:
    print('0/1')
elif dot == 1:
    print('1/6')
elif dot == 2:
    print('1/3')
elif dot == 3:
    print('1/2')
elif dot == 4:
    print('2/3')
elif dot == 5:
    print('5/6')
elif dot == 6:
    print('1/1')

File: Problemas/9A - Die Roll/9A_Die_Roll.cpp
// 9A - Die Roll 
// https://codeforces.com/problemset/problem/9/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) yakko = Resultado del dado de Yakko
    // 2) wakko = Resultado del dado de Wakko
    int yakko, wakko;
    cin >> yakko >> wakko;

    // Calcula el valor del dado que Dot debe superar
    int valor_maximo = max(yakko, wakko);

    // Cantidad de valores que le darán a Dot la victoria
    int dot = 6 - valor_maximo + 1;

    // Simplifica la fracción acorde a las especificaciones del problema
    switch(dot) {
        case 0: cout << "0/1" << endl; break;
        case 1: cout << "1/6" << endl; break;
        case 2: cout << "1/3" << endl; break;
        case 3: cout << "1/2" << endl; break;
        case 4: cout << "2/3" << endl; break;
        case 5: cout << "5/6" << endl; break;
        case 6: cout << "1/1" << endl; break;
    }

    return 0;
}